config/izum_vega.py

```diff
@@ -59,47 +59,57 @@
-                    ],
-                    'resources': [
-                        {
-                            'name': '_rfm_gpu',
-                            'options': ['--gpus-per-node={num_gpus_per_node}'],
-                        },
-                        {
-                            'name': 'memory',
-                            'options': ['--mem={size}'],
-                        }
-                    ],
-                    'features': [
-                        FEATURES[GPU],
-                    ] + list(SCALES.keys()),
-                    'descr': 'GPU partition, see https://en-doc.vega.izum.si/architecture/'
-                },
+#                 {
+#                     'name': 'gpu',
+#                     'scheduler': 'slurm',
+#                     'prepare_cmds': [
+#                         'source %s' % common_eessi_init(),
+#                         # Pass job environment variables like $PATH, etc., into job steps
+#                         'export SLURM_EXPORT_ENV=ALL',
+#                         # Needed when using srun launcher
+#                         # 'export SLURM_MPI_TYPE=pmix',  # WARNING: this broke the GROMACS on Vega
+#                         # Avoid https://github.com/EESSI/software-layer/issues/136
+#                         # Can be taken out once we don't care about old OpenMPI versions anymore (pre-4.1.1)
+#                         'export OMPI_MCA_pml=ucx',
+#                     ],
+#                     'launcher': 'mpirun',
+#                     # Use --export=None to avoid that login environment is passed down to submitted jobs
+#                     'access': ['-p gpu', '--export=None'],
+#                     'environs': ['default'],
+#                     'max_jobs': 60,
+#                     'devices': [
+#                         {
+#                             'type': DEVICE_TYPES[GPU],
+#                             'num_devices': 4,
+#                         }
+#                     ],
+#                     'resources': [
+#                         {
+#                             'name': '_rfm_gpu',
+#                             'options': ['--gpus-per-node={num_gpus_per_node}'],
+#                         },
+#                         {
+#                             'name': 'memory',
+#                             'options': ['--mem={size}'],
+#                         }
+#                     ],
+#                     'features': [
+#                         FEATURES[GPU],
+#                     ] + list(SCALES.keys()),
+#                     'extras': {
+#                         # Make sure to round down, otherwise a job might ask for more mem than is available
+#                         # per node
+#                         'mem_per_node': 476.837  # in GiB (should be checked, its unclear from slurm.conf)
+#                     },
+#                     'descr': 'GPU partition, see https://en-doc.vega.izum.si/architecture/'
+#                 },
```

Comment:
```suggestion
{
                #     'name': 'gpu',
                #     'scheduler': 'slurm',
                #     'prepare_cmds': [
                #         'source %s' % common_eessi_init(),
                #         # Pass job environment variables like $PATH, etc., into job steps
                #         'export SLURM_EXPORT_ENV=ALL',
                #         # Needed when using srun launcher
                #         # 'export SLURM_MPI_TYPE=pmix',  # WARNING: this broke the GROMACS on Vega
                #         # Avoid https://github.com/EESSI/software-layer/issues/136
                #         # Can be taken out once we don't care about old OpenMPI versions anymore (pre-4.1.1)
                #         'export OMPI_MCA_pml=ucx',
                #     ],
                #     'launcher': 'mpirun',
                #     # Use --export=None to avoid that login environment is passed down to submitted jobs
                #     'access': ['-p gpu', '--export=None'],
                #     'environs': ['default'],
                #     'max_jobs': 60,
                #     'devices': [
                #         {
                #             'type': DEVICE_TYPES[GPU],
                #             'num_devices': 4,
                #         }
                #     ],
                #     'resources': [
                #         {
                #             'name': '_rfm_gpu',
                #             'options': ['--gpus-per-node={num_gpus_per_node}'],
                #         },
                #         {
                #             'name': 'memory',
                #             'options': ['--mem={size}'],
                #         }
                #     ],
                #     'features': [
                #         FEATURES[GPU],
                #     ] + list(SCALES.keys()),
                #     'extras': {
                #         # Make sure to round down, otherwise a job might ask for more mem than is available
                #         # per node
                #         'mem_per_node': 476.837  # in GiB (should be checked, its unclear from slurm.conf)
                #     },
                #     'descr': 'GPU partition, see https://en-doc.vega.izum.si/architecture/'
                # },
```